waspc/data/Generator/templates/react-app/vite.config.ts

```diff
@@ -1,21 +1,34 @@
+{{={= =}=}}
 /// <reference types="vitest" />
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react-swc'
+import { mergeConfig } from "vite";
+import react from "@vitejs/plugin-react-swc";
 
-// https://vitejs.dev/config/
-export default defineConfig({
+{=# isCustomViteConfigUsed =}
+import _waspUserProvidedConfig from "./src/ext-src/vite.config";
```

Comment:
Mhm this is very hardcoded here, can we instead pass this path from Haskell and make it less hardcoded? I think it should not be very hard to do?

Reply:
Yep, I would generate a `JsImport` in Haskell and pass it in here 👍